EpiAware/src/EpiInfModels/EpiInfModels.jl

```diff
@@ -10,7 +10,7 @@ using ..EpiAwareUtils: scan, censored_pmf
 using Turing, Distributions, DocStringExtensions, LinearAlgebra
 
 #Export models
-export EpiData, DirectInfections, ExpGrowthRate, Renewal
+export EpiData, DirectInfections, ExpGrowthRate, Renewal, RenewalWithPopulation
```

Comment:
Maybe it should be `PopulationBoundedRenewal`?

Reply:
I feel thats a super-type if/when we start adding more models.